apps/snitch_core/lib/core/data/model/product.ex

```diff
@@ -511,4 +511,8 @@ defmodule Snitch.Data.Model.Product do
   def is_variant_tracking_enabled?(product) do
     Product.is_variant_tracking_enabled?(product)
   end
+
+  def upn_generate() do
+    "A" <> Nanoid.generate(10, "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ") <> "C"
```

Comment:
@jyotigautam I thought we only agreed on numbers in between the `A` & `C`. Did I miss something?

Reply:
change the logic here.